scripts/build-data.js

```diff
@@ -115,27 +116,47 @@ const getLowestImplementedVersion = ({ features }, env) => {
   .reduce((a, b) => { return (a < b) ? b : a; });
 };
 
-const data = {};
-for (const pluginName in pluginFeatures) {
-  const options = pluginFeatures[pluginName];
-  const plugin = {};
-  environments.forEach((env) => {
-    if (Array.isArray(options.features)) {
+function generateData(features) {
+  let ret = {};
+
+  Object.keys(features).forEach((pluginName) => {
+    let options = features[pluginName];
+
+    if (!options.features) {
+      options = {
+        features: [options]
+      };
+    }
+
+    const plugin = {};
+    environments.forEach((env) => {
       const version = getLowestImplementedVersion(options, env);
       if (version !== null) {
         plugin[env] = version;
       }
-    }
 
-    // add opera
-    if (plugin.chrome) {
-      plugin.opera = plugin.chrome - 13;
-    }
+      // add opera
+      if (plugin.chrome) {
+        plugin.opera = plugin.chrome - 13;
+
+        if (plugin.chrome === 5) {
+          plugin.opera = 12;
```

Comment:
Because of how the statement in line 140 sets opera to chrome - 13, you might get opera -12, opera -11, ..., opera 14 (assuming there is data for chrome older than 28) :X